tests/spell.test.ts

```diff
@@ -20,4 +20,20 @@ describe('Test summoner spells fetching.', () => {
     const byName = await client.summonerSpells.findByName('Flash');
     expect(byName).toBe(flash);
   });
+
+  test('Check spell caching', async () => {
+    const cachedFlash = client.summonerSpells.cache.find((s) => s.id === 'SummonerFlash')!;
+    expect(cachedFlash.name).toBe(flash.name);
+  });
+
+  test('Check spells pre-fetching', async () => {
+    const client2 = new Client(process.env.riot_api_key!);
+    await client2.initialize({
+      cache: false,
+      fetch: {
+        summonerSpells: true
+      }
+    });
+    expect(client2.summonerSpells.cache.size).toBeGreaterThan(10);
```

Comment:
Similar to rune pages, these are also likely to stay 11 for the foreseeable future, better to make this an exact check.